src/lazy/expanded/compiler.rs

```diff
@@ -460,11 +460,9 @@ impl TemplateCompiler {
         match module_name {
             // If the module is `$ion`, this refers to the system module.
             "$ion" => ION_1_1_SYSTEM_MACROS.clone_macro_with_id(macro_id),
-            // If the module is `$ion_encoding`, this refers to the active encoding module.
-            "$ion_encoding" => context.macro_table().clone_macro_with_id(macro_id),
-            _ => todo!(
-                "qualified references to modules other than $ion_encoding (found {module_name}"
-            ),
+            // If the module is `_`, this refers to the active encoding module.
+            "_" => context.macro_table().clone_macro_with_id(macro_id),
```

Comment:
It would be good to have a `pub const DEFAULT_MODULE_NAME` for this so that we can use a named constant instead of a string.